### PR TITLE
fix(wallet)_: error on tapping from/to section in tx confirmation screen

### LINKED ISSUE #21693: Error "Cannot set property 'reagent-compiler2' of null" to confirmation page on send flow
# Bug Report

### Reproduction

1. Start the Send Flow.
2. Select your account as the recipient (navigate to the "My Accounts" tab).
3. Choose Ethereum as the asset.
4. Enter an amount to send.
5. Click the Review Send button.
6. On the confirmation page, tap the header or the fields displaying sender/recipient information (From and To).

#### Expected behavior
The header and fields (From and To) should not be clickable and should not trigger any actions or errors.

#### Actual behavior
Clicking the header or the fields (From and To) triggers an error popup  text: "Cannot set property 'reagent-compiler2' of null".
https://github.com/user-attachments/assets/7db8bedf-b3bf-4b5f-abde-e2548da481da

### Additional Information

- Status version: nightly  2.30.0 (20241126020846)
- Operating System: Android, iOS

Log from Android: [Status-debug-logs (1).zip](https://github.com/user-attachments/files/17933468/Status-debug-logs.1.zip)

### PR DESCRIPTION
**Pull request by smohamedjavid**
fixes #21693

### Summary

This PR fixes the error thrown when tapping the From or To sections in the transaction confirmation screen.

### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Navigate to the Wallet tab
- Go to the send flow
- Choose an asset and an account or enter an address to send
- Enter an amount and generate a route
- Navigate to transaction confirmation
- Verify the app is not throwing any error when tapping the From or To sections

status: ready

### CODE REVIEW

File: src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs [hunk 1]
```diff
@@ -282,37 +282,33 @@
            :gradient-cover?          true
            :customization-color      (:color account)}
           [rn/view
-           [rn/pressable
-            {:on-press #(rf/dispatch [:navigate-to-within-stack
-                                      [:screen/wallet.transaction-details
-                                       :screen/wallet.transaction-confirmation]])}
```
Comment (smohamedjavid) [inline src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs:292]: @vkjr I believe this `on-press` is not needed as there is no screen transition required from the confirmation page, and we do not have a `wallet.transaction-details` screen. This is introduced in https://github.com/status-im/status-mobile/pull/21657. Let me know if I missed anything.

**Review by briansztamfater — APPROVED** [review]
LGTM 👍

**Review by shivekkhurana — APPROVED** [review]

**Review by clauxx — APPROVED** [review]

File: src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs (hunk 1 above)
Comment (vkjr) [reply src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs:292]: @smohamedjavid, wow, thanks for noticing! That is a leftover that I made for developer convenience while worked on mentioned PR but forgot to remove 🤦

**Review by vkjr — APPROVED** [review]

**Comment by pavloburykh** [comment]
@smohamedjavid thanks for the PR. Ready for merge.

**Pull request merged**